packages/components/pie-button/src/index.ts

```diff
@@ -37,9 +36,8 @@ export class PieButton extends LitElement {
         };
 
         const raiseWCEvent = () => {
-            const event = new CustomEvent('CustomEvent', { detail: 'WC event dispatched' })
             console.info('WC event dispatched')
-            this.dispatchEvent(event)
+            this.dispatchEvent(new CustomEvent('onCustomEvent', { detail: 'WC event dispatched' }))
```

Comment:
just my personal preference but I think defining the event before dispatching it is a bit more readable

Reply:
I don't think the event should be called `onCustomEvent`. That sounds more like a handler function that would respond to `CustomEvent`. What's the need for the change?

Reply:
I agree with the `onCustomEvent` naming, it should be separate to the handler - I've reverted that back and updated the wrapper to match 👍

In terms of the dispatch, unfortunately defining the event before dispatching means the event isn't read by the `custom-elements-manifest` so isn't added to the `custom-elements.json`